.github/workflows/documentation.yml

```diff
@@ -0,0 +1,25 @@
+name: Documentation
+
+on:
+  push:
+    branches:    
+      - master         # Push events on master branch
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        python-version: 3.7
+    - name: Run documentation script
+      run: |
```

Comment:
Would keep the actions syntax. organization:

```
- name: install dependencies
  run: |
    python -m pip install --upgrade pip 
    pip install sphinx doctr

- name: build
  run: make html
  working_directory: docs

- name: Deploy documentation
   run: doctr deploy . --built-docs docs/_build/html --key-path .deploy_key.enc
```

Reply:
I think your comment is outdated

Reply:
It's not, pip and make in same part, and actions uses `working_directory` not `cd`.  Also `actions-gh-pages` doesn't to use doctr so doesn't need to be installed.

Reply:
Sorry this one was outdated yes